step.yml

```diff
@@ -26,8 +26,13 @@ inputs:
   - plist_path:
     opts:
       title: "Info.plist file path"
+      summary: Path to the given target's Info.plist file. You need to use this step for each archivable target of your project.
       description: |
-        Path to the given project's Info.plist file.
+        **NOTE:**<br/>
+        If your IPA contains multiple build targets ( e.g extensions ), they would need to have the same version number just like your app main target's.<br/>
+        You need to add this step to your workflow for every build target one by one.
+
+        If there are targets with different version numbers in the exported IPA, Apple will reject it.
```

Comment:
This description also would be great in the step's description, even if the wf editor collapses it, on the integration's page it is visible by default and this description is more generally applicable for the whole step then to this input only.

Reply:
If there are targets with different version numbers the app cannot be submitted for App Review or Beta App Review.

Reply:
Fixed